codecs/vp8_packet.go

```diff
@@ -127,12 +127,11 @@ func (p *VP8Packet) Unmarshal(payload []byte) ([]byte, error) {
 
 	payloadLen := len(payload)
 
-	if payloadLen < 4 {
-		return nil, errShortPacket
-	}
-
 	payloadIndex := 0
 
+	if payloadIndex >= payloadLen {
```

Comment:
Just a readability thing here, given that `payloadIndex` has just been set to 0, IMO this would be easier to grok as
`if payloadLen == 0` as this is the only case you're really guarding against (i.e. `payloadIndex` can't be greater than `payloadLen` so that leaves only the equality case, and they'd only be equal if `payloadLen` was 0).